### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by gxdgxd** [gtamas/etcdmanager#78]
Could not perform operation. Error: Error: 8 RESOURCE_EXHAUSTED: Received message larger than max (8222366 vs. 4194304)

**Comment by arthurzenika** [comment]
Same bug here

![Screenshot from 2021-05-20 13-50-32](https://user-images.githubusercontent.com/445200/118974035-a32d5700-b972-11eb-943d-923bbcfdeab3.png)

On a etcd database extracted from rancher/kubernetes (the db in members/snap is 200Mb)

**Comment by gtamas** [comment]
Yeah, that's because of how this is implemented. We have to refactor that part.

So I know what's the problems I have plans for fixing this already, but currently I just don't have a lot of time for this project.
I'm in the middle of moving to a different continent. However, I will get back to this as soon as I can.

In the meantime, PRs are welcome :) If someone wants to help out, please let me know! I'm happy to provide guidance regarding the current code.

**Comment by arthurzenika** [comment]
@gtamas hey! thanks for replying! Good look on your move. No rush on my part, just making sure issues are reported so other users don't get the same bugs.

**Comment by lfarkas** [comment]
duplicate of #123
